Week1/databases/newversion.js

```diff
@@ -0,0 +1,79 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'ghufran',
+  password: 'deiri',
+  database: 'userdb',
+});
+
+connection.connect();
+
+const cb = function(error, results, fields) {
+  if (error) {
+    throw error;
+  }
+};
+
+connection.query('CREATE DATABASE IF NOT EXISTS world', cb());
+
+connection.query('use world', cb());
+
+connection.query(
+  ` CREATE TABLE IF NOT EXISTS country (
+    Code CHAR(3) NOT NULL DEFAULT '',
```

Comment:
like that the definition is detailed and split into different lines, makes it much nicer to read 👍

Reply:
I think I love DATABASES :)